src/usage/schema.md

```diff
@@ -0,0 +1,7 @@
+# Schema
+
+Did you land here with cryptic errors? Then please install the following official GitHub Actions VSCode Extension https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions and review all errors/warnings shown there.

```

Comment:
```suggestion
Did you land here because of cryptic errors? Then please install the following official GitHub Actions VSCode Extension https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions and review all errors/warnings shown by it.

```